ddtrace/contrib/internal/langchain/patch.py

```diff
@@ -1101,6 +1190,8 @@ def patch():
         )
         wrap("langchain_core", "runnables.base.RunnableSequence.batch", traced_lcel_runnable_sequence(langchain))
         wrap("langchain_core", "runnables.base.RunnableSequence.abatch", traced_lcel_runnable_sequence_async(langchain))
+        wrap("langchain_core", "tools.BaseTool.invoke", traced_base_tool_invoke(langchain))
+        wrap("langchain_core", "tools.BaseTool.ainvoke", traced_base_tool_ainvoke(langchain))
```

Comment:
Looks like older versions of langchain-core (`v0.1.x`) are failing this as the `tools` module is not accessible directly through `langchain_core`. I suggest adding this line at line 1171:
`from langchain_core.tools import BaseTool  # noqa:F401`
This lets us import the necessary module/class before wrapping it.